jenkins-build-k8s-snaps.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+KUBE_VERSION=${KUBE_VERSION:-"v1.5.2"}
+
+git clone https://github.com/juju-solutions/release.git --branch rye/snaps
```

Comment:
Hey as long as we're checking out a branch directly, let's make it a shallow clone with `--depth 1`